db/migrate/20180129133617_create_work_schedules.rb

```diff
@@ -0,0 +1,24 @@
+class CreateWorkSchedules < ActiveRecord::Migration[5.1]
```

Comment:
Let's just fully remove these migrations then if we don't use these tables - makes it easier to run them again.

Reply:
Hm... Currently, I have a migration to replace these with the array. Should I throw them all out and start over in that regard?

Reply:
Yep. Just get the `schema.rb` from `master`, remove the migration files we don't need and adjust the others to just add the 2 columns and everything's nice and clean.

I can also quickly do it - no more need for more work for you. You did an awesome job already.